test/iOS.test.ts

```diff
@@ -17,10 +30,19 @@ describe('Test for iOS platform', () => {
     expect(iosVersion).toBe('>= 1.2.3 and < 4.5.6')
   })
 
+  it('returns correct version with dsl', async () => {
```

Comment:
The CI runner that checks coverage diff doesn't have the `pod` command installed, which is causing the failure. One option is to **temporarily** use `it.skip` as a workaround. Another option is to add pod support to the runner based on a parameter (since it's also used in other projects). A third option would be to create an `itif / it_if` function that runs the test only if `pod` is available, and skips it otherwise (this approach is not recommended). The last option is mocking `pod` command, but I'm not sure about that.

Reply:
Fixed by introducing param to configure runner in the shared workflow: https://github.com/fingerprintjs/dx-team-toolkit/pull/118